resources/public/github-btn.html

```diff
@@ -1 +1 @@
co{background-position:-14px 0}.gh-count{position:relative;display:none;margin-left:4px;background-color:#fafafa;border:1px solid #d4d4d4}.gh-count:hover,.gh-count:focus{color:#4183C4}.gh-count:before,.gh-count:after{content:'';position:absolute;display:inline-block;width:0;height:0;border-color:transparent;border-style:solid}.gh-count:before{top:50%;left:-3px;margin-top:-4px;border-width:4px 4px 4px 0;border-right-color:#fafafa}.gh-count:after{top:50%;left:-4px;z-index:-1;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#d4d4d4}.github-btn-large{height:30px}.github-btn-large .gh-btn,.github-btn-large .gh-count{padding:3px 10px 3px 8px;font-size:16px;line-height:22px;border-radius:4px}.github-btn-large .gh-ico{width:20px;height:20px;background-size:40px 20px}.github-btn-large .gh-btn:hover .gh-ico,.github-btn-large .gh-btn:focus .gh-ico,.github-btn-large .gh-btn:active .gh-ico{background-position:-20px 0}.github-btn-large .gh-count{margin-left:6px}.github-btn-large .gh-count:before{left:-5px;margin-top:-6px;border-width:6px 6px 6px 0}.github-btn-large .gh-count:after{left:-6px;margin-top:-7px;border-width:7px 7px 7px 0}</style><span class=github-btn id=github-btn><a class=gh-btn id=gh-btn href=# target=_blank><span class=gh-ico></span> <span class=gh-text id=gh-text></span></a> <a class=gh-count id=gh-count href=# target=_blank></a></span><script>function addCommas(t){return String(t).replace(/(\d)(?=(\d{3})+$)/g,"$1,")}function jsonp(t){var e=document.createElement("script");e.src=t+"?callback=callback",head.insertBefore(e,head.firstChild)}function callback(t){"watch"==type?counter.innerHTML=addCommas(t.data.watchers):"fork"==type?counter.innerHTML=addCommas(t.data.forks):"follow"==type&&(counter.innerHTML=addCommas(t.data.followers)),"true"==count&&(counter.style.display="block")}var params=function(){for(var t,e=[],o=window.location.href.slice(window.location.href.indexOf("?")+1).split("&"),r=0;r<o.length;r++)t=o[r].split("="),e.push(t[0]),e[t[0]]=t[1];return e}(),user=params.user,repo=params.repo,type=params.type,count=params.count,size=params.size,head=document.getElementsByTagName("head")[0],button=document.getElementById("gh-btn"),mainButton=document.getElementById("github-btn"),text=document.getElementById("gh-text"),counter=document.getElementById("gh-count");button.href="https://github.com/"+user+"/"+repo+"/","watch"==type?(mainButton.className+=" github-watchers",text.innerHTML="Star",counter.href="https://github.com/"+user+"/"+repo+"/stargazers"):"fork"==type?(mainButton.className+=" github-forks",text.innerHTML="Fork",counter.href="https://github.com/"+user+"/"+repo+"/network"):"follow"==type&&(mainButton.className+=" github-me",text.innerHTML="Follow @"+user,button.href="https://github.com/"+user,counter.href="https://github.com/"+user+"/followers"),"large"==size&&(mainButton.className+=" github-btn-large"),jsonp("follow"==type?"https://api.github.com/users/"+user:"https://api.github.com/repos/"+user+"/"+repo);</script>
```

Comment:
the change here is

`:root {color-scheme: light dark} body{background-color: light-dark(#f8f8f8,#333);`

to make the github button iframe also respect Dark mode